### DISCUSSION TITLE
geo: URI aka RFC 5870 support?

### DISCUSSION THREAD

**Discussion by l29ah** [osm-search/Nominatim#3207] [Ideas]
https://en.wikipedia.org/wiki/Geo_URI_scheme

Would be great if nominatim could "find" these.

**Comment by mtmail** [comment]
"geo:37.786971,-122.399677" Since the Nominatim API needs to be called anyway, can't the caller split the text? Is there an example of any other geocoding program that supports these?

**Reply by l29ah** [reply]
osmand, marble, google maps, for example.

**Reply by mtmail** [reply]
What should the output of Nominatim be? Nominatim is an API, the map on nominatim.osm.org is for debugging purposes. I see the www.openstreetmap.org website had this feature requests multiple times and isn't implementing it https://github.com/openstreetmap/openstreetmap-website/issues/1123
